### ISSUE TITLE
Error: Failed to load user: Error: Failed to fetch key: users/xxx: File not found

### ISSUE THREAD

**Issue by wwaechter** [jhuckaby/Cronicle#770]
I get the following error by opening "Admin"-tab: Error: Failed to load user: Error: Failed to fetch key: users/xxx: File not found.
The error appears after I've deleted a test user account.

## Steps to reproduce the problem
1. I've created (chronicle GUI) a test account for PEN testing. 
2. The test user have tested the application: created and running events, changing password and so on.
3. After the test I've delted (chronicle GUI) the test account.
Since the deleting I've got the error. 

## Your Setup

docker container on kubernetes

### Operating system and version?

alpine

### Node.js version?
v20.13.1

### Cronicle software version?
0.9.51

### Are you using a multi-server setup, or just a single server?
single server
### Are you using the filesystem as back-end storage, or S3/Couchbase?
filesystem
### Can you reproduce the crash consistently?
yes
## Log Excerpts
Error from User.log: 
[1717744685.179][2024-06-07 09:18:05][xxx][20][User][error][user][Failed to load user: Error: Failed to fetch key: users/xxx: File not found][]

**Comment by jhuckaby** [comment]
How strange, I cannot reproduce this bug.  Could be a transient browser caching issue maybe.  If you refresh the page, does the error go away?

**Comment by wwaechter** [comment]
After I've copied the user folder from another instance. The access to the Admin-Tab working again. Additinally I've created and deleted the user. Now it works as it should. 
May be it was a special situation: I've deleted the user with the version 0.9.39, after it I've deployed a new docker image with th the version 0.9.51. I belive it was the state of the data in the container and on the permanent storage.
